src/app/@ansyn/imagery/communicator-service/communicator.entity.ts

```diff
@@ -347,4 +357,16 @@ export class CommunicatorEntity implements OnInit, OnDestroy {
 		}
 	}
 
+	private calcBoundingRect() {
+		const boundingRect = this.ActiveMap.getHtmlContainer().getBoundingClientRect();
+		// cut the status bar and the menu
+		return {
+			x: Math.round(boundingRect.left - 90),
```

Comment:
what do 90 and 55 mean

Reply:
// cut the status bar and the menu

Reply:
are those represent the sizes of the status bar and the menu?